src/transformers/models/layoutlm/modeling_tf_layoutlm.py

```diff
@@ -934,10 +936,10 @@ def call(
         Examples:
 
         ```python
-        >>> from transformers import LayoutLMTokenizer, TFLayoutLMModel
+        >>> from transformers import AutoTokenizer, TFLayoutLMModel
```

Comment:
Could you also remove _TOKENIZER_FOR_DOC = "LayoutLMTokenizer" in modeling_tf_layoutlm.py, since it's not being used?

Reply:
Yeah